README.md

```diff
@@ -199,6 +213,22 @@ $ lukso start --lighthouse --erigon
 $ lukso start --logs-folder "[folder_path]"
 ```
 
+#### Using Checkpoint Syncing
```

Comment:
I think we can mention that `Checkpoint sync is a feature that significantly speeds up the initial sync between your beacon node and the Beacon Chain. With checkpoint sync configured, your beacon node will begin syncing from a recently finalized checkpoint instead of syncing from genesis. This can make installations, validator migrations, recoveries, and testnet deployments way faster.` here